### ISSUE TITLE
Selected (aria-selected=true) state is not announced for selected options inside a listbox

### ISSUE THREAD

**Issue by dobrinyonkov** [nvaccess/nvda#12893]
### Steps to reproduce:
1. open https://cookiecrook.com/test/aria/multiselect/listbox.html
2. select some of the options
3. start NVDA
4. navigate to last option - using arrow down key
5. navigate to first option - using arrow up key

### Actual behavior:
Upon navigation, Selected (aria-selected=true) state is not announced for selected options. 
Only Not Selected (aria-selected=false) is announced.

### Expected behavior:
Selected (aria-selected=true) state to be announced for all selected options. 

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2021.2
#### Windows version:
Version	10.0.19042 Build 19042
#### Name and version of other software in use when reproducing the issue:
Chrome Version 94.0.4606.61 (Official Build) (64-bit)
#### Does the issue still occur after restarting your computer? 
YES

#### Have you tried any other versions of NVDA? If so, please report their behaviors. 
2020.*

#### If NVDA add-ons are disabled, is your problem still occurring? 
YES

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu? 
YES

**Comment by feerrenrut** [comment]
Only announcing one of the states (selected vs not selected) reduces verbosity, the other state is implied.
When considering this behavior in the context of a standard listbox (no multi select) this makes a lot more sense, the item focused IS selected, the user doesn't have to hear selected / vs not selected at all.

The UX for multiselect list box is extending this convention.

Note, in the example, checking the dev tools (accessibility tab) shows `multiselectable = false`. Adding `aria-multiselectable="true"` fixes this. However, I don't believe this changes the behavior of NVDA.

See also https://github.com/nvaccess/nvda/issues/4816

A case could be made that this behavior is confusing, especially for new users. If you'd like to discuss this side of it more I can re-open.
